metaflow/plugins/aws/awslambda/lambda_cli.py

```diff
@@ -0,0 +1,204 @@
+import os
+import sys
+
+import boto3
+import click
+import traceback
+
+from botocore.config import Config
+from metaflow import R, util
+from metaflow.exception import METAFLOW_EXIT_DISALLOW_RETRY
+from metaflow.metaflow_config import (
+    LAMBDA_IMAGE_URI,
+    LAMBDA_ROLE_ARN,
+    LAMBDA_THROTTLE_RETRIES,
+)
+from metaflow.plugins.aws.utils import sync_metadata_from_s3, TaskSpec
+
+from .lambda_runner import LambdaRunner, ensure_lambda, LambdaRuntimeException
+from metaflow.mflog import TASK_LOG_SOURCE
+
+if sys.version_info > (3, 0):
+    from typing import Dict, Optional
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(help="Commands related to AWS Lambda.")
+def awslambda():
+    pass
+
+
+@awslambda.command(
+    help="Execute a single task using AWS Lambda. This command "
+    "calls the top-level step command inside a AWS Lambda "
+    "job with the given options. Typically you do not "
+    "call this command directly; it is used internally "
+    "by Metaflow."
+)
+@click.argument("step-name")
+@click.argument("code-package-sha")
+@click.argument("code-package-url")
+@click.option("--memory", type=int, help="Memory requirement for AWS Lambda.")
+@click.option("--run-id", help="Passed to the top-level 'step'.")
+@click.option("--task-id", help="Passed to the top-level 'step'.")
+@click.option("--image", help="Lambda image")
+@click.option(
```

Comment:
I don't think `force_fetch_image` is needed any longer.

Reply:
Discussed offline -- will leave it in but as an advanced feature that we won't advertise. Normally users are expected to build an image under a new tag when it changes

Reply:
..and rename to `refresh_image`